lib/diagnostic.js

```diff
@@ -46,21 +46,31 @@ let person2 = {
 };
 
 // Question 3:  Invoke the `fullName()` method so it returns `Santa Clause`?
-let answer3 = null; // replace null with your response
+let answer3 = {
+  fullname: function() {
+    return person2.firstName + " " + person2.lastName;
+  }
+}; // replace null with your response
 
 // Question 4
 
-let alderaan = { name: 'Alderaan' };
+let alderaan = {
+  name: 'Alderaan'
+};
 let deathStar = {
   fire: function() {
     return this.name;
-   }
+  }
 };
 
 // Question 4: Write the code that would set the `deathStar`'s `fire` method
 // in the context of `alderaan` but does not invoke the method,
 // so that we could call answer4() // Alderaan
-let answer4 = null;  // replace null with your response
+let answer4 = function() {
```

Comment:
This answer misses the point of the exercise.  `deathStar`'s `fire()` method needs to be bound to `alderaan` so that it's used as the context.  This is done by calling `Function`'s `bind()` on `fire()` with `alderaan` as the argument.

```javascript
deathStar.fire.bind(alderaan);
```